docs/standard/using-linq.md

```diff
@@ -155,7 +155,7 @@ The answer to this question is **no** if...
 
 ## Essential Samples
 
-For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](https://code.msdn.microsoft.com/101-LINQ-Samples-3fb9811b).
+For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](/samples/dotnet/try-samples/101-linq-samples/).
```

Comment:
I was confused why the /dotnet came after the /samples but now I realize this is not a link to the /dotnet/samples repo. In that case, it should be a fully qualified link since the main samples repo that you're linking to isn't part of offline books, and leaving off the https://docs.microsoft.com is only to make the link work in offline books.

```suggestion
For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](https://docs.microsoft.com/samples/dotnet/try-samples/101-linq-samples/).
```

Reply:
Awesome, thank you!